uint16 pedal_position
uint8 counter
int8 position_diff
